.gitmodules

```diff
@@ -1,10 +1,10 @@
 [submodule "modules/pvData"]
     path = modules/pvData
-    url = https://github.com/epics-base/pvDataCPP
+    url = https://github.com/mdavidsaver/pvDataCPP
     branch = master
 [submodule "modules/pvAccess"]
     path = modules/pvAccess
-    url = https://github.com/epics-base/pvAccessCPP
+    url = https://github.com/mdavidsaver/pvAccessCPP
```

Comment:
Should we keep this as epics-base?

Reply:
If the submodule revisions are now in main repo (I think there are), then these can be switched back.